.prettierrc

```diff
@@ -1,4 +1,9 @@
 {
+  "trailingComma": "es5",
```

Comment:
Not going to implement 'all'. Just 'es5' is fine. The rationale behind trailing comma at the end of multi lines object or array is that when we're adding a new line and commit, the only line that'll be shown as changed is only the new implementation, and not the previous line simply because we added another comma to that line.